CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 
 ## v0.9.2 
+- update Memento dependency version
```

Comment:
This should be in the staged category, and would get moved only during a new release

Reply:
Addressed.